products/knownprojects/sql/aggregate/sca/boundaries_school_subdistricts.sql

```diff
@@ -338,7 +338,7 @@ FROM (
         b.proportion_in_subdist_1 AS proportion_in_subdist,
         round(a.units_net * b.proportion_in_subdist_1) AS units_net_in_subdist
     FROM
-        _kpdb AS a
+        kpdb AS a
```

Comment:
Huh. so seems like these aggregate tables were using not the final table before as their input?